server/graphql/v1/types.js

```diff
@@ -658,6 +658,18 @@ export const UpdateType = new GraphQLObjectType({
           return update.image;
         },
       },
+      isPrivate: {
+        type: GraphQLBoolean,
+        resolve(update) {
+          return update.isPrivate;
+        },
+      },
+      isContentAvailable: {
```

Comment:
`isContentAvailable` doesn't seems accurate to me if this is about permission. Maybe `isAllowedToSee`, or `userCanSeeUpdate` would be better. Whether you want to keep the actual or prefer to change, there should be a `description` on the field to be explicit about what it does.

Reply:
I've renamed 'isContentAvailable' to 'userCanSeeUpdate', and added a description.